crates/db_views/src/combined/report_combined_view.rs

```diff
@@ -392,6 +402,33 @@ impl ReportCombinedQuery {
   }
 }
 
+// Only show reports for communities where the user is a mod
+#[diesel::dsl::auto_type]
+fn filter_is_mod() -> _ {
+  community_actions::became_moderator
+    .is_not_null()
+    .and(report_combined::community_report_id.is_null())
+}
+
+// Filter reports which are only for admins
+#[diesel::dsl::auto_type]
+fn filter_violates_instance_rules() -> _ {
+  post_report::violates_instance_rules
+    .or(comment_report::violates_instance_rules)
+    .or(community_report::id.is_not_null())
+    .or(private_message_report::id.is_not_null())
+}
+
+// Admins can see reports intended for them, or mod reports older than 3 days
+#[diesel::dsl::auto_type]
+fn filter_visible_to_admins(interval: DateTime<Utc>) -> _ {
+  filter_violates_instance_rules()
+    .or(filter_is_mod())
```

Comment:
Why do you need this is_mod filter here? It shouldn't need to filter out anything if you're the admin.